### DISCUSSION TITLE
Is there a way to force GPU-use in CLI ?

### DISCUSSION THREAD

**Discussion by Mamacito473** [Yutaka-Sawada/MultiPar#121] [Q&A] [answered]
Hey,

First of all, thank you really much Yutaka for your tool. I have a noobish question for you. I'm using your tool with NGpost,

Right now I use par2j.exe with these parameters `PAR2_ARGS = create /rr20 /lc80 /lr /rd2 /ss768000`

is there a way to use the GPU instead of CPU ? I guess using GPU will speed up things to beyond imaginable since I have a 4090 GPU. 

Any tips for me ? thank you rly much

**Comment by Yutaka-Sawada** [comment, marked as answer]
To enable GPU acceleration, set `/lc256` option in the command-line now. (In old versions, it was `/lc32` or `/lc64`.) For details, you may refer manual of par2j; "Command_par2j.txt" in "help" folder of MultiPar. Or, you may read [it online at github](https://github.com/Yutaka-Sawada/MultiPar/blob/master/alpha/help/Command_par2j.txt).
